airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -164,16 +164,18 @@ def check_for_bucket(self, bucket_name: Optional[str] = None) -> bool:
             return False
 
     @provide_bucket_name
-    def get_bucket(self, bucket_name: Optional[str] = None) -> str:
+    def get_bucket(self, bucket_name: Optional[str] = None, *, region_name: Optional[str] = None) -> str:
         """
         Returns a boto3.S3.Bucket object
 
         :param bucket_name: the name of the bucket
         :type bucket_name: str
         :return: the bucket object to the bucket name.
         :rtype: boto3.S3.Bucket
+        :param region_name: The name of the aws region in which to get the bucket.
+        :type region_name: str
         """
-        session, endpoint_url = self._get_credentials()
+        session, endpoint_url = self._get_credentials(region_name=region_name)
```

Comment:
It didn't have region_name provided before (even before my PR). Are we changing the behavior altogether now? This sounds like a fix but for completely different unrelated problem :)

Reply:
I felt it’s apretty cheap feature to add. But you’re right, let’s not complicate things. We can always add those back is someone wants them.